packages/flutter_tools/test/general.shard/migrate/migrate_utils_test.dart

```diff
@@ -0,0 +1,203 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
I probably shouldn't review this test file further until the other changes have been made. Let me know when I should take another look!